packit/fedpkg.py

```diff
@@ -81,12 +83,29 @@ def build(
             cmd += ["--target", koji_target]
         if srpm_path:
             cmd += ["--srpm", str(srpm_path)]
-        utils.run_command_remote(
-            cmd=cmd,
-            cwd=self.directory,
-            error_message="Submission of build to koji failed.",
-            fail=True,
-        )
+
+        try:
+            utils.run_command_remote(
+                cmd=cmd,
+                cwd=self.directory,
+                error_message="Submission of build to koji failed.",
+                fail=True,
+            )
+
+        except PackitCommandFailedError as ex:
+            # fail on the fedpkg side, the build is triggered
+            if (
+                "watch_tasks() got an unexpected keyword argument 'ki_handler'"
+                in ex.stderr_output
+            ):
+                logger.info(
+                    "fedpkg build command crashed which is a known issue: "
+                    "the build is submitted in koji anyway"
+                )
+                logger.debug(ex.stdout_output)
+
+            else:
+                raise ex
```

Comment:
just `raise` (without the `ex`) is enough

Reply:
changed